src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -1264,7 +1264,7 @@ public void mouseMoved(final Territory territory, final MouseDetails me) {
             // is the only one for
             // which the route may actually change much)
             if (unitsThatCanMoveOnRoute.size() < selectedUnits.size() && (unitsThatCanMoveOnRoute.size() == 0
-                || Match.allMatchNotEmpty(unitsThatCanMoveOnRoute, Matches.UnitIsAir))) {
+                || !unitsThatCanMoveOnRoute.isEmpty() && Match.allMatch(unitsThatCanMoveOnRoute, Matches.UnitIsAir))) {
```

Comment:
Parentheses are not required here, but it may make it easier to read due to the `||`.